playbooks/setup_node_exporter_rpi.yml

```diff
@@ -0,0 +1,59 @@
+---
+- name: setup node exporter on rpis
+  hosts: raspberry
+
+  tasks:
+    - name: If armv6 architecture, download and unpack node_exporter for armv6 from prometheus
+      ansible.builtin.unarchive:
+        src: https://github.com/prometheus/node_exporter/releases/download/v1.1.2/node_exporter-1.1.2.linux-armv6.tar.gz
+        dest: /home/pi/
+        remote_src: true
+      become: true
+      when: ansible_facts['architecture'] is search("armv6")
+
+    - name: If armv6 architecture, copy node_exporter to /usr/local/bin/
+      ansible.builtin.copy:
```

Comment:
Do you think that there is a way to merge the two task in a single one ? Namely unarchive and copy. Would it be possible to unarchive directly in the wanted directory ? 

That would be nicer !